src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message 'Excavator started its work.' is not very informative. Consider specifying what kind of work the Excavator is doing to make the message more informative.